test_common/harness/kernelHelpers.cpp

```diff
@@ -1369,8 +1369,8 @@ test_status verifyImageSupport( cl_device_id device )
 {
     if( checkForImageSupport( device ) )
     {
-        log_error( "ERROR: Device does not supported images as required by this test!\n" );
-        return TEST_FAIL;
+        log_error( "SKIPPED: Device does not supported images as required by this test!\n" );
+        return TEST_SKIP;
```

Comment:
This doesn't quite work. With the code as is you'd end up skipping the test if the query in `checkForImageSupport` wasn't successful. I suggest testing for `CL_IMAGE_FORMAT_NOT_SUPPORTED` explicitly:
- 0 => `TEST_PASS`
- `CL_IMAGE_FORMAT_NOT_SUPPORTED` => `TEST_SKIP`
- else => `TEST_FAIL`

Reply:
Okay, I'll make these changes.